src/triggers/TDTM_AttendanceEvent.trigger-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexTrigger xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>46.0</apiVersion>
```

Comment:
This is still 46.0 but I talked to @lifewithryan about this today and I think it's okay to leave as-is.